docs/operator/kubernetes-cluster-monitoring/README.md

```diff
@@ -0,0 +1,16 @@
+## quick start guide for setup cluster components monitoring
```

Comment:
This whole file does not look finished - no useful info in it

Reply:
@f41gh7 anything we can add here or remove completely?

Reply:
I'd vote to remove it

Reply:
remote it, i'll update doc generator